<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>schedule</class>
 <widget class="QDialog" name="schedule">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>FitApp - Упражнения</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Элементы управления</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отменить изменения</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Сохранить изменения</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="DeleteButton">
       <property name="text">
        <string>Удалить выделенные записи</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Элементы добавления</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Сложность</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="difficultyComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Название упражнения</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="exerciseNameLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Группа мыщц</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="muscleGroupComboBox"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Добавить упражнение</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="authButton">
       <property name="text">
        <string>Вернуться к авторизации</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="timetableButton">
       <property name="text">
        <string>Расписание</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setButton">
       <property name="text">
        <string>Создать сет</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
